components/logins/src/store.rs

```diff
@@ -154,7 +153,7 @@ impl PasswordStore {
     }
 
     pub fn check_valid_with_no_dupes(&self, login: &Login) -> Result<()> {
-        self.db.check_valid_with_no_dupes(login)
+        self.db.check_valid_with_no_dupes(&login)
```

Comment:
Maybe I'm reading the rust wrong, but is this change needed?  It seems like one more level of referencing than required.